pgtype/float4.go

```diff
@@ -195,3 +196,27 @@ func (src *Float4) Value() (driver.Value, error) {
 		return nil, errUndefined
 	}
 }
+
+func (src *Float4) MarshalJSON() ([]byte, error) {
+	switch src.Status {
+	case Present:
+		buf := make([]byte, 4)
+		binary.BigEndian.PutUint32(buf, math.Float32bits(src.Float))
+		return buf, nil
```

Comment:
I don't think this is right. Shouldn't this return the text of the number, not the raw bits?